migration-engine/connectors/migration-connector/src/migrations_directory.rs

```diff
@@ -3,11 +3,11 @@
 //! Migrations directory management.
 //!
 //! This module is responsible for the management of the contents of the
-//! migrations directory. The migrations directory contains multiple migration
-//! directories, named after the migration id, and each containing:
-//!
+//! migrations directory. At the top level it contains a schema.lock file which lists the provider.
```

Comment:
nitpick: the constant below says the file name is `migration_lock.toml`

Reply:
Well spotted. I'll change it later to allow for quicker merging now